extensions/hibernate-reactive/deployment/src/test/java/io/quarkus/hibernate/reactive/NoJtaTest.java

```diff
@@ -29,21 +25,12 @@ public class NoJtaTest {
                     .addClass(MyEntity.class)
                     .addAsResource("application.properties"));
 
-    @Inject
-    SessionFactory sessionFactory; // This is an ORM SessionFactory, but it's backing Hibernate Reactive.
-
     @Inject
     Mutiny.SessionFactory factory;
 
     @Test
     @RunOnVertxContext
     public void test(UniAsserter asserter) {
-        ServiceRegistryImplementor serviceRegistry = sessionFactory.unwrap(SessionFactoryImplementor.class)
-                .getServiceRegistry();
-
-        // Two assertions are necessary, because these values are influenced by separate configuration
-        assertThat(serviceRegistry.getService(JtaPlatform.class).retrieveTransactionManager()).isNull();
-        assertThat(serviceRegistry.getService(TransactionCoordinatorBuilder.class).isJta()).isFalse();
```

Comment:
The test is called `NoJtaTest`, and we're no longer testing there's no JTA... :]